layout_padded/layout_padded.md

```diff
@@ -70,7 +70,7 @@ Revision 2 to be submitted 2023-01-15.
 
   * Simplify `layout_{left,right}_padded::mapping` constructor conditions
 
-  * Add default constructors for `layout_{left,right}_padded::mapping`; `=default` for _`actual-padding-stride`_` != dynamic_extent`, not `=default` otherwise
+  * Add default constructors for `layout_{left,right}_padded::mapping`; `=default` for _`static-padding-stride`_` != dynamic_extent`, not `=default` otherwise
```

Comment:
If this was a new change, should we consider adding it instead to the R4 list (which doesn't exist yet, but will)?